platform/solr/solr-factory-impl/src/main/java/org/codice/solr/factory/impl/ConfigurationStore.java

```diff
@@ -42,7 +43,7 @@
    *
    * <p>To circumvent the bug that this one place creates when it starts re-configuring the store to
    * use an in-memory Solr server, we were forced to make this constructor public. The {@link
-   * EmbeddedSolrFactory}Solr factory class was properly modified to rely on the passed in
+   * SolrClientFactory}Solr factory class was properly modified to rely on the passed in
```

Comment:
✏️ This statement might no longer be true. It was not the SolrClientFactory but really the embedded one as originally stated.

Reply:
So is that not necessary anymore now that the embeddedsolrprovider has been removed? Can I just remove this constructor?